spotbugs-exclude.xml

```diff
@@ -36,4 +36,8 @@
             <Class name="com.github.fppt.jedismock.operations.strings.SetEx"/>
         </Or>
     </Match>
+    <Match>
```

Comment:
What is the reason for this?

Reply:
otherwise spotbugs execution [fails](https://github.com/fppt/jedis-mock/actions/runs/11763330493/job/32776997557):
```
[INFO] --- spotbugs-maven-plugin:4.8.6.5:check (check) @ jedis-mock ---
[INFO] BugInstance size is 5
[INFO] Error size is 0
[INFO] Total bugs: 5
Error:  Medium: com.github.fppt.jedismock.operations.scripting.LuaCjsonLib inherits equals and uses Object.hashCode() [com.github.fppt.jedismock.operations.scripting.LuaCjsonLib] At LuaCjsonLib.java:[lines 20-29] HE_INHERITS_EQUALS_USE_HASHCODE
Error:  Medium: com.github.fppt.jedismock.operations.scripting.LuaCjsonLib$decode inherits equals and uses Object.hashCode() [com.github.fppt.jedismock.operations.scripting.LuaCjsonLib$decode] At LuaCjsonLib.java:[lines 109-178] HE_INHERITS_EQUALS_USE_HASHCODE
Error:  Low: The class name com.github.fppt.jedismock.operations.scripting.LuaCjsonLib$decode doesn't start with an upper case letter [com.github.fppt.jedismock.operations.scripting.LuaCjsonLib$decode] At LuaCjsonLib.java:[lines 109-178] NM_CLASS_NAMING_CONVENTION
Error:  Medium: com.github.fppt.jedismock.operations.scripting.LuaCjsonLib$encode inherits equals and uses Object.hashCode() [com.github.fppt.jedismock.operations.scripting.LuaCjsonLib$encode] At LuaCjsonLib.java:[lines 32-105] HE_INHERITS_EQUALS_USE_HASHCODE
Error:  Low: The class name com.github.fppt.jedismock.operations.scripting.LuaCjsonLib$encode doesn't start with an upper case letter [com.github.fppt.jedismock.operations.scripting.LuaCjsonLib$encode] At LuaCjsonLib.java:[lines 32-105] NM_CLASS_NAMING_CONVENTION
```

`#equals` comes from `LuaValue` class and its implementation is equalient to default one: 
```java
	// object equality, used for key comparison
	public boolean equals(Object obj)         { return this == obj; } 
```